docs/install/ubuntu_setup.md

```diff
@@ -161,24 +161,24 @@ For other libraries, visit the [Math Library Selection](build_from_source.html#m
 If building on CPU and using OpenBLAS:
 
 ```bash
-    git clone --recursive https://github.com/apache/incubator-mxnet.git
-    cd mxnet
+    git clone --recursive https://github.com/apache/incubator-mxnet.git ~/mxnet
+    cd ~/mxnet
```

Comment:
Could we maybe rename this to incubator-mxnet instead of making use of an absolute path (in the users home directory)?
The resulting line would be ``cd incubator-mxnet``

Reply:
ok